R/extractData2.R

```diff
@@ -165,11 +189,11 @@ labels2values2 <- function(dat, labels, convertMiss, dropPartialLabels, labels2c
   changed_variables_labels2factor <- intersect(labels2factor, changed_variables)
   changed_variables <- setdiff(changed_variables, changed_variables_labels2factor)
   if(length(changed_variables_labels2factor) > 0) {
-    dat2 <- char2fac(dat = dat2, labels = labels, vars = changed_variables_labels2factor, convertMiss = convertMiss, ordered = FALSE)
+    dat2 <- char2fac(dat = dat2, labels = change_labels, vars = changed_variables_labels2factor, convertMiss = convertMiss, ordered = FALSE)
```

Comment:
I received an error message at this point:
```
Fehler in char2fac(dat, labels = gads$labels, vars = "v1", convertMiss = TRUE) : 
  konnte Funktion "char2fac" nicht finden
```
It seems that `char2fac` isn't exported to the NAMESPACE? (`only multiChar2fac`)

Reply:
`char2fac()` is an internal function and can be found in `R/extractData.R`. I know that these two syntaxes are currently more complex and disorganized than they could/should be, which I plan to adress in #82.